applications/app/views/fragments/galleryHeader.scala.html

```diff
@@ -0,0 +1,93 @@
+@(item: model.ContentType, showBadge: Boolean = false, showMeta: Boolean = true)(implicit request: RequestHeader)
+
+@import views.support.TrailCssClasses.toneClass
+@import views.support.ContributorLinks
+
+<header class="content__head content__head--immersive is-fixed-height tonal__head tonal__head--@toneClass(item)">
+
+    <style>
+        @defining(
+            item.content.elements.elements.head
+        ) { picture =>
+            .content__header--background-image {
+                background-image: url('http:@Html(ImgSrc.findNearestSrc(picture.images, Profile(width = Some(1000))).getOrElse(""))');
+            }
+            @@media (min-width: @{layout.Phablet.minWidth}px) {
+                .content__header--background-image {
+                    background-image: url('http:@Html(ImgSrc.findNearestSrc(picture.images, Profile(width = Some(1300))).getOrElse(""))');
+                }
+            }
+            @@media (min-width: @{layout.Wide.minWidth}px) {
+                .content__header--background-image {
+                    background-image: url('http:@Html(ImgSrc.findNearestSrc(picture.images, Profile(width = Some(2200))).getOrElse(""))');
+                }
+            }
```

Comment:
FWIW I did use object fit for 100vh images in the minute: https://github.com/guardian/frontend/pull/11985/files#diff-c557d31de28ab1c0964b852829df0b8fR548